### ISSUE TITLE
bug: failed to run playground mode using docker image

### ISSUE THREAD

**Issue by mikechesterwang** [risingwavelabs/risingwave#7238]
### Describe the bug

failed to `docker run risingwavelabs/risingwave:v0.1.15 playground`, seems RisingWave requires a config file located in `src/config/default.toml`.

### To Reproduce

run 
```bash
 docker run -p 4566:4566 -p 5691:5691 risingwavelabs/risingwave:v0.1.15 playground
```
got
```
launching playground
config file: src/config/default.toml
config file: src/config/default.toml
config file: src/config/default.toml
thread 'main' panicked at 'failed to open config file 'src/config/default.toml': No such file or directory (os error 2)', src/common/src/config.rs:39:29
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

a workaround:
```
docker run --entrypoint "/usr/bin/bash" \
risingwavelabs/risingwave:v0.1.15 \
-c "mkdir -p /src/config && touch /src/config/default.toml && /risingwave/bin/risingwave playground"
```

### Expected behavior

`docker run risingwavelabs/risingwave:v0.1.15 playground` can work without workarounds.

**Comment by xxchan** [comment]
It is ok when `risedev.yml` and the config file are both present, or both absent. 

In the docker we have `risedev.yml` but not the config file, so it panics. Actually caused by this change https://github.com/risingwavelabs/risingwave/pull/6814/files#diff-e1cea435bf1e8e5ef4692d74e955179ec3f3579c6e5e596d51849fe89c54b8acL68

**Comment by mikechesterwang** [comment]
So the workaournd is mouting a risedev.yml to the image. Not so user-friendly🤔

**Comment by xxchan** [comment]
I'm thinking whether we should decouple playground with risedev.. And only use hard-coded config for playground instead... (pr introducing playground with risedev config https://github.com/risingwavelabs/risingwave/pull/2097)

**Comment by mikechesterwang** [comment]
I think this is fixed by https://github.com/risingwavelabs/risingwave/pull/7223

**Comment by xxchan** [comment]
> I think this is fixed by #7223

No, will become a different missing file "failed to open config file 'src/config/default.toml'"
